galaxy_ng/app/signals/handlers.py

```diff
@@ -22,6 +22,19 @@ def ensure_retain_repo_versions_on_repository(sender, instance, created, **kwarg
         instance.save()
 
 
+@receiver(post_save, sender=AnsibleDistribution)
+def ensure_content_guard_exists_on_distribution(sender, instance, created, **kwargs):
+    """Ensure distribution have a content guard when created."""
+
+    content_guard, _ = ContentRedirectContentGuard.objects.get_or_create(
```

Comment:
I think you can just do `objects.first()` here. We can trust that the migrations created the content guard successfully.